src/steamship/data/tags/tag_constants.py

```diff
@@ -1,8 +1,21 @@
 from typing import Optional
 
 
+class TagKind:
+    """A set of `kind` constants for Tags."""
+
+    pos = "pos"  # Part of speech tags
+    dep = "dep"  # Dependency tags
+    sentiment = "sentiment"  # Sentiment tags
+    emotion = "emotion"  # Emotion tags
+    ent = "ent"  # Entity tags, such as PERSON or ORG
```

Comment:
```suggestion
    ent = "entity"  # Entity tags, such as PERSON or ORG
```

I prefer full name vs abbreviations as they require less documentation.

Reply:
Agree. Longer names don't cost us anything.